api/tasks/runner_test.go

```diff
@@ -23,6 +23,11 @@ func TestCheckTmpDir(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	if runtime.GOOS == "windows" {
+		// Windows doesn't support os.Chmod, need use acl.Chmod
+		return
```

Comment:
rather than just skipping the tests on windows can you rewrite them to work, or remove this change and make a new issue about it so we can work on it separately?

Thanks!

Reply:
I think this test is incorrect. It tests limitations of the file system. Maybe remove it?

Reply:
This test will fail under Linux if Temp directory located on NTFS partition for example.

Reply:
well it tests if the checkTmpDir will return an error or not if the folder you try to create has a read only parent, as this needs to bubble up to us if we do it by accident. Maybe there is a better way to test it, but we still need to test it, as part of this functions job is to let us know if making the dir was a success or not

Reply:
I've replaced OS checking to checking ```os.Chmod``` support. But I still think that this test should be removed because it is not portable.

Reply:
I'll take that for now but in the end it should be fixed so it is portable, because we need to check that our function returns false if we cannot create the directory. 

If you can think of a better and more portable way to check for this then i will happily accept it :smile: basically the condition should be that the function is not allowed to write into the directory that we attempt to create a folder in and this should return an error.